### DISCUSSION TITLE
No way to perform an update after pressing "Skip"

### DISCUSSION THREAD

**Discussion by tannisroot** [home-assistant/frontend#12402] [Dashboard] [answered]
If you accidentally press "Skip" when presented an update message, there's no way to cancel that to see the update message again and thus perform the update through the UI, it only seems possible through CLI.
That seems like a major issue in UX, but I was told this does not count as a bug and should be filed as a feature request instead.

**Comment by spacegaier** [comment, marked as answer]
This will be resolved with the next upcoming version which will include the option to display skipped updates.
